src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -317,8 +318,8 @@ private async Task OrchestrationMiddleware(DispatchMiddlewareContext dispatchCon
 
             FunctionName orchestratorFunction = new FunctionName(context.Name);
 
-            OrchestratorInfo info;
-            if (!this.registeredOrchestrators.TryGetValue(orchestratorFunction, out info))
+            RegisteredFunctionInfo info;
+            if (!this.knownOrchestrators.TryGetValue(orchestratorFunction, out info))

```

Comment:
Because we are no longer throwing an exception here for deregistered orchestrators, we will try executing them below. Again, I am assuming this is intentional, but I'm a little confused as to what "deregistered" means at this point.

Reply:
You are correct. Yes, the meaning of "deregistered" is a little confusing now. The only reason we need to keep track of this status is so that we do the right thing in `StopTaskHubWorkerIfIdleAsync` - basically shut down the task hub worker when all triggers are "deregistered". I'm happy to use a different property name if you have good ideas. :)